certification/runtime/config.go

```diff
@@ -51,24 +52,36 @@ func NewConfigFrom(vcfg viper.Viper) (*Config, error) {
 	cfg.DockerConfig = vcfg.GetString("dockerConfig")
 	cfg.Artifacts = vcfg.GetString("artifacts")
 	cfg.WriteJUnit = vcfg.GetBool("junit")
-	cfg.storeContainerPolicyConfiguration(vcfg)
 	cfg.storeOperatorPolicyConfiguration(vcfg)
+
+	err := cfg.storeContainerPolicyConfiguration(vcfg)
+	if err != nil {
+		return nil, err
+	}
+
 	return &cfg, nil
 }
 
 // storeContainerPolicyConfiguration reads container-policy-specific config
 // items in viper, normalizes them, and stores them in Config.
-func (c *Config) storeContainerPolicyConfiguration(vcfg viper.Viper) {
+func (c *Config) storeContainerPolicyConfiguration(vcfg viper.Viper) error {
 	c.PyxisAPIToken = vcfg.GetString("pyxis_api_token")
 	c.Submit = vcfg.GetBool("submit")
 	c.PyxisHost = pyxisHostLookup(vcfg.GetString("pyxis_env"), vcfg.GetString("pyxis_host"))
 
 	// Strip the ospid- prefix from the project ID if provided.
 	certificationProjectID := vcfg.GetString("certification_project_id")
 	if strings.HasPrefix(certificationProjectID, "ospid-") {
```

Comment:
Does it make more sense to do these validations _before_ we get here? Like as a `PreRunE`. 

I can see it both ways; it won't take much to sway me one way or the other. Consider this non-blocking. Just thinking out loud.

Reply:
I don't think that the flags would be loaded into viper if there was a `PreRunE`, but I might be wrong.

@bcrochet Thoughts on this?

Reply:
You can add a PreRunE. It should still run the PersistentPreRun.

Reply:
The flags do get loaded before `PreRunE` so I have removed the logic from `config.go` to `check_containter.go` PTAL.